### DISCUSSION TITLE
Simulating Alkaline AA Batteries

### DISCUSSION THREAD

**Discussion by WadeRPX** [pybamm-team/PyBaMM#2693] [Q&A]
Is pybamm able to simulate disposable Alkaline AA batteries? How should I go about using pybamm to simulate discharging AA batteries?

**Comment by valentinsulzer** [comment]
Not out of the box. It would probably be a large project to set up (e.g. masters project)

**Reply by WadeRPX** [reply]
Thank you for such a speedy response.
